### PR TITLE
Always set user defaults on RepoHandler

### PR DESCRIPTION
**Pull request by M-Davies**
* Reduces context to a case by case basis
* Also tests to see if USER_DEFAULTS_JSON is working properly (with debug out in the class)

Closes: #133 
Signed-off-by: Morgan Davies <morgandavies2020@gmail.com>

### CODE REVIEW

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
run tests

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -1061,23 +1062,25 @@ class Build {
                                 updateGithubCommitStatus("PENDING", "Build Started")
                             }
                             if (useAdoptShellScripts) {
-                                context.println "[CHECKOUT] Checking out to AdoptOpenJDK/openjdk-build to use their shell scripts..."
-                                repoHandler.checkoutAdoptBuild()
+                                context.println "[CHECKOUT] Checking out to AdoptOpenJDK/openjdk-build..."
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy:1065]: should that change to Adoptium temurin-build?

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**